app/src/main/java/com/amaze/filemanager/filesystem/files/FileUtils.java

```diff
@@ -404,7 +406,7 @@ public static void installApk(
   }
 
   private static void openUnknownInternal(
-      Uri contentUri, String type, Context c, boolean forcechooser, boolean useNewStack) {
+      Uri contentUri, String type, MainActivity c, boolean forcechooser, boolean useNewStack) {
```

Comment:
Why is Context now Activity?

Reply:
I guess various underlying methods / static open file fragment calls need it.